docs/_includes/topbar.html

```diff
@@ -16,10 +16,14 @@
                     <a class="nav-link cw-nav-link cw-header-link-text" href="news.html">News</a>
                 </li>
                 
-                <li class="nav-item cw-navbar-item cw-header-link-news">
+                <li class="nav-item cw-navbar-item cw-header-link-blog">
```

Comment:
Will the "blog" tab replace the "news" tab? I thought we wanted both "blogs" and "news" as separate tabs.